packages/beeq/src/components/textarea/bq-textarea.tsx

```diff
@@ -314,6 +339,45 @@ export class BqTextarea {
     this.hasHelperText = hasSlotContent(this.helperTextElem);
   };
 
+  private setFormValue = (value: string) => {
+    console.log('setFormValue', value);
+    const textareaValue = value ? value : '';
+    this.internals?.setFormValue(textareaValue, `${this.value}`);
```

Comment:
Please remove the `console.log()` and it is better to use undefined if no value has been set, that way the component won't participate in the form submission.

```suggestion
    this.internals?.setFormValue(textareaValue, value ?? undefined);
```